### DISCUSSION TITLE
If I start the Sexbound animation I only have one animation to choose from (sit on face).

### DISCUSSION THREAD

**Discussion by zylonen** [ErinaSugino/Starbound-SBR-Mod#78] [Trouble Shooting] [answered]
I have a problem. When I start the sex animation I only  have one animation "Sit on face". I cannot choose another position.
I have installed everything properly on your mod description. The only time i got to choose all animations was during the Aphrodites Bow Quest.
Any ideas what could be the issue ?
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/13757258/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
Even without looking into the log I would assume that you are playing a female character and currently interact with a female entity. In that case only that position would be available.

**Reply by zylonen** [reply]
Yes thats correct. 
Thank you for your fast reply <3
Have a nice christmas :)
